src/styles/components/journal-card.css

```
@@ -0,0 +1,48 @@
+.journal-card__wrapper {
+  display: grid;
+  grid-gap: 20px;
+}
+
+.journal-card {
+  box-shadow: 0px 2px 11px 0 var(--blue-shadow);
+  padding: 20px;
+}
+
+.journal-card__heading--bg {
+  color: var(--blue);
+  font-size: 14px;
+  text-transform: uppercase;
+  line-height: 18px;
+}
```

```suggestion
font-weight: 600;
}
```
Hier sollte noch die Font Weight rein?